resources/kernel/make_kernel_image_deb.sh

```diff
@@ -18,7 +19,12 @@ source "$TEST_RESOURCE_DIR/make_kernel.sh"
 
 KERNEL_VERSION="5.4.81"
 
-KERNEL_CFG="microvm-kernel-5.4-x86_64.config"
+if [[ $arch = "x86_64" ]]; then
+	KERNEL_CFG="microvm-kernel-5.4-x86_64.config"
```

Comment:
I think you can construct KERNEL_CFG based on the arch value, instead of having this if else.

Reply:
there are some changes that was required in the default config.
earlier also a pre made config file was present